rpc/Cargo.toml

```diff
@@ -47,7 +47,7 @@ ethcore-miner = { path = "../miner" }
 ethcore-private-tx = { path = "../ethcore/private-tx" }
 ethcore-sync = { path = "../ethcore/sync" }
 ethcore-transaction = { path = "../ethcore/transaction" }
-ethereum-types = "0.3"
+ethereum-types = "0.3.2"
```

Comment:
Grumble, if we should keep `ethereum-types = "0.3"` here!

Reply:
We need at least 0.3.2 to include the fix. How about `~0.3.2`?

Reply:
`0.3.2` is the same as `~0.3.2`

Reply:
You're right (`>=0.3.2 <0.4`). So it's just narrowing the range to include the version with the fix.